src/index.css

```diff
@@ -55,3 +55,84 @@
 .keypadBtns .keypadBtn:last-child {
   border-left: none;
 }
+
+/* header section */
+
+.header {
+  display: flex;
+  width: 100%;
+  height: 10vh;
+  background: orangered; /* darkslategray */
+}
+
+.header .nav {
+  display: flex;
+  width: 100%;
+  max-width: 950px;
+  height: 90%;
+  justify-content: space-between;
+  align-self: center;
+  margin: 0 auto;
+}
```

Comment:
- [ ] Kindly remove the `max-width` from these styles as it's not making your design match with the wireframe given.

**Your design**
![image](https://user-images.githubusercontent.com/50941074/172010807-cb4f7284-1dab-4acc-b89b-c5f028a77e06.png)

**The Wireframe**
![image](https://user-images.githubusercontent.com/50941074/172010869-6148cbd3-0f9d-41c1-8226-dfebd7738655.png)

- [ ] Kindly add `padding: 0 40px;` to the .header .nav styles and see how things look for the header. 👍 
Adding the padding and removing the max-width will give you a design that looks like the screenshot below.

![image](https://user-images.githubusercontent.com/50941074/172011080-0d5fe5ec-96fc-4a02-837a-0ad2ec222add.png)

Reply:
Removed max-width and added suggested change